ext/gd/gd.stub.php

```diff
@@ -10,59 +10,60 @@ function gd_info(): array {}
+function imagesetstyle(GdImage $image, array $style): bool {}
 
-function imagecreatetruecolor(int $x_size, int $y_size): GdImage|false {}
+function imagecreatetruecolor(int $width, int $height): GdImage|false {}
 
-function imageistruecolor(GdImage $im): bool {}
+function imageistruecolor(GdImage $image): bool {}
 
-function imagetruecolortopalette(GdImage $im, bool $ditherFlag, int $colorWanted): bool {}
+function imagetruecolortopalette(GdImage $image, bool $dither, int $num_colors): bool {}
 
-function imagepalettetotruecolor(GdImage $im): bool {}
+function imagepalettetotruecolor(GdImage $image): bool {}
 
-function imagecolormatch(GdImage $im1, GdImage $im2): bool {}
+function imagecolormatch(GdImage $image1, GdImage $image2): bool {}
 
-function imagesetthickness(GdImage $im, int $thickness): bool {}
+function imagesetthickness(GdImage $image, int $thickness): bool {}
 
-function imagefilledellipse(GdImage $im, int $cx, int $cy, int $w, int $h, int $color): bool {}
+function imagefilledellipse(GdImage $image, int $center_x, int $center_y, int $width, int $height, int $color): bool {}
 
-function imagefilledarc(GdImage $im, int $cx, int $cy, int $w, int $h, int $s, int $e, int $col, int $style): bool {}
+function imagefilledarc(GdImage $image, int $center_x, int $center_y, int $width, int $height, int $start_angle, int $end_angle, int $color, int $style): bool {}
 
-function imagealphablending(GdImage $im, bool $blend): bool {}
+function imagealphablending(GdImage $image, bool $enable): bool {}
 
-function imagesavealpha(GdImage $im, bool $save): bool {}
+function imagesavealpha(GdImage $image, bool $enable): bool {}
 
-function imagelayereffect(GdImage $im, int $effect): bool {}
+function imagelayereffect(GdImage $image, int $effect): bool {}
 
-function imagecolorallocatealpha(GdImage $im, int $red, int $green, int $blue, int $alpha): int|false {}
+function imagecolorallocatealpha(GdImage $image, int $red, int $green, int $blue, int $alpha): int|false {}
 
-function imagecolorresolvealpha(GdImage $im, int $red, int $green, int $blue, int $alpha): int|false {}
+function imagecolorresolvealpha(GdImage $image, int $red, int $green, int $blue, int $alpha): int|false {}
 
-function imagecolorclosestalpha(GdImage $im, int $red, int $green, int $blue, int $alpha): int|false {}
+function imagecolorclosestalpha(GdImage $image, int $red, int $green, int $blue, int $alpha): int|false {}
 
-function imagecolorexactalpha(GdImage $im, int $red, int $green, int $blue, int $alpha): int|false {}
+function imagecolorexactalpha(GdImage $image, int $red, int $green, int $blue, int $alpha): int|false {}
 
-function imagecopyresampled(GdImage $dst_im, GdImage $src_im, int $dst_x, int $dst_y, int $src_x, int $src_y, int $dst_w, int $dst_h, int $src_w, int $src_h): bool {}
+function imagecopyresampled(GdImage $dst_image, GdImage $src_image, int $dst_x, int $dst_y, int $src_x, int $src_y, int $dst_width, int $dst_height, int $src_width, int $src_height): bool {}
```

Comment:
I think we have used copy with from/to so far, so I think you could switch the `$dst`/`$src` prefixes without making the signature significantly longer.

Reply:
Hmm, I find `$from_x` etc. somewhat confusing. OTOH, this is a legacy function with a terrible signature anyway; one is likely better off using `imagescale()` (plus `imagecrop()` if needed).

Reply:
Ok, so this is something we shouldn't waste too much brain cycles ^^

Reply:
Yeah, I also don't like how $from/$to look into context.